### PR TITLE
Update custom-server.md

### PR DESCRIPTION
**Pull request by bartduisters**
Corrected position of some sentences

### CODE REVIEW

File: docs/guides/custom-server.md [hunk 1]
```diff
@@ -129,8 +131,8 @@ keystone
 
 ## Custom Server with manual middleware preparation
 
-For really fine-grained control, a custom server skip calling
-`keystone.prepare()` in favour of calling an app's `.prepareMiddleware()`
+For really fine-grained control, a custom server skips calling
```
Comment (MadeByMike) [inline docs/guides/custom-server.md]:
```suggestion
For really fine-grained control, a custom server can skip calling
```

File: docs/guides/custom-server.md [hunk 2]
```diff
@@ -129,8 +131,8 @@ keystone
...
+`keystone.prepare()` in favor of calling an app's `.prepareMiddleware()`
```
Comment (MadeByMike) [inline docs/guides/custom-server.md]:
```suggestion
`keystone.prepare()` in favour of calling an app's `.prepareMiddleware()`
```

File: docs/guides/custom-server.md [hunk 3]
```diff
@@ -15,20 +15,22 @@ handles the GraphQL API and Admin UI. Things such as:
 - Add additional routes
 - Setup additional server middleware (`compress`/`brotli`/etc)
 - Notify a 3rd party service when the API is ready
-- ... etc
+- ...
 
 A **Custom Server** can replace the default and act as the entry point to your
 application which consumes your [schema definition](/docs/guides/schema.md). A Custom
-Server must handle initialising a http server which correctly executes any given KeystoneJS Apps.
+Server must handle initializing a http server which correctly executes any given KeystoneJS Apps.
```
Comment (MadeByMike) [inline docs/guides/custom-server.md]:
```suggestion
Server must handle initialising a http server which correctly executes any given KeystoneJS Apps.
```

Comment (bartduisters) [reply docs/guides/custom-server.md]: I noticed, while reading the documentation, that indeed most was en-UK. New PRs will not contain corrections to en-US..

**Pull request merged**